src/clib/lib/job_queue/job_node.cpp

```diff
@@ -281,7 +281,7 @@ job_queue_node_alloc(const char *job_name, const char *run_path,
     node->sim_start = 0;
     node->sim_end = 0;
     node->submit_time = time(NULL);
-    node->max_confirm_wait = 60 * 2; // 2 minutes before we consider job dead.
+    node->max_confirm_wait = 60 * 10; // 2 minutes before we consider job dead.
```

Comment:
comment still valid after this change?

Reply:
haha lol obviously not XD sorry, and thanks for catching that ^^

Reply:
fixed!